### DISCUSSION TITLE
Add README to the "@tanstack/react-query-next-experimental" NPM page

### DISCUSSION THREAD

**Discussion by jacob-leger** [TanStack/query#5770] [Ideas] [answered]
# Add README to the `@tanstack/react-query-next-experimental` NPM page

## What?? 🤷‍♂️

As you will notice in the image below, there is no README.md file for the `@tanstack/react-query-next-experimental` NPM package

<img width="494" alt="no-readme-in-tanstack-query" src="https://github.com/TanStack/query/assets/95201376/79f6acf4-d4be-463d-9e32-527f4da86c67">

## Where?? 🏛️

[npmjs.com/package/@tanstack/react-query-next-experimental/](https://www.npmjs.com/package/@tanstack/react-query-next-experimental)

## What do I propose then?? 📋

I propose that someone (me, someone on the team, or anyone who would like to help) will at least create a short README file for the [experimental package's NPM page](npmjs.com/package/@tanstack/react-query-next-experimental). I don't know what the Tanstack team would like best, but even a short description explaining that the package is in the alpha or beta stages or whatever stage it's in.

I would be happy to write the README, as I am proficient in Markdown, but I have no idea what the Tanstack team would want me to put in there.

If you are reading this and you are part of the Tanstack or react-query team, please reach out and tell me how I could help!

**Comment by TkDodo** [comment, marked as answer]
yeah please add a readme :)

**Reply by jacob-leger** [reply]
I would love to... But I am not really an expert on tanstack-query.

Could you explain to me where this file would be located
I also am confused on what this package does - not tanstack-query but `@tanstack/react-query-next-experimental`. I also do not know if there is any specific style that your team uses for their READMEs.

BTW I live your blog articles!

**Reply by jacob-leger** [reply]
Is this package still in use? I'm sorry, but I think I forgot all about it.

~~Should I just close the discussion as outdated?~~

Update: It looks like the package was updated recently, and it's frequently getting worked on. If someone could tell me what this package was for, I could help with the README.

**Reply by shreecodes** [reply]
> If someone could tell me what this package was for, I could help with the README.

The use case for this package is documented here:

https://tanstack.com/query/latest/docs/framework/react/guides/advanced-ssr#experimental-streaming-without-prefetching-in-nextjs
